src/redturtle/importer/volto/transmogrifier/types_mapping.py

```diff
@@ -61,6 +61,12 @@ def __call__(self, item, typekey):
             item["_defaultitem"] = ""
             item["default_page"] = ""
             return item
+        elif portal_type == "News Item":
```

Comment:
questa è una modifica che serve solo per design.plone.contenttypes, non in generale